### PR TITLE
Add Deity Boons/Curses from the Gods and Magic Web Supplement... again

### PR DESCRIPTION
**Pull request by LebombJames**
Accidentally managed to delete the original branch this was on, but through some fumbling around git I managed to salvage my changes... thank fuck for version control.

### CODE REVIEW

File: packs/data/boons-and-curses.db/grandmother-spider-moderate-boon.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Grandmother Spider - Moderate Boon",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/grandmother-spider-moderate-boon.json:20]:
```suggestion
```

File: packs/data/boons-and-curses.db/besmara-moderate-boon.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Besmara - Moderate Boon (Initiative while on a boat)",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/besmara-moderate-boon.json:20]:
```suggestion
```

File: packs/data/boons-and-curses.db/besmara-moderate-boon.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                        "terrain:boat"
+                    ]
+                },
+                "selector": "initiative",
+                "type": "status",
+                "value": 2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Besmara - Moderate Boon (Acrobatics to Balance while on a boat)",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/besmara-moderate-boon.json:32]:
```suggestion
```

File: packs/data/boons-and-curses.db/besmara-moderate-boon.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                        "terrain:boat"
+                    ]
+                },
+                "selector": "acrobatics",
+                "type": "status",
+                "value": 2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Besmara - Moderate Boon (Athletics to Climb while on a boat)",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/besmara-moderate-boon.json:45]:
```suggestion
```

File: packs/data/boons-and-curses.db/casandalee-minor-curse.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Casandalee",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/casandalee-minor-curse.json:20]:
```suggestion
```

File: packs/data/boons-and-curses.db/grandmother-spider-minor-curse.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "adjustment": {
+                    "failure": "one-degree-worse"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "label": "Grandmother Spider - Minor Curse",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/grandmother-spider-minor-curse.json:23]:
```suggestion
```

File: packs/data/boons-and-curses.db/grandmother-spider-minor-curse.json [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                "selector": "stealth",
+                "type": "skill"
+            },
+            {
+                "adjustment": {
+                    "failure": "one-degree-worse"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "label": "Grandmother Spider - Minor Curse",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/grandmother-spider-minor-curse.json:32]:
```suggestion
```

File: packs/data/boons-and-curses.db/gruhastha-major-boon.json [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "adjustment": {
+                    "success": "one-degree-better"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "label": "Gruhastha - Major Boon",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/gruhastha-major-boon.json:23]:
```suggestion
```

File: packs/data/boons-and-curses.db/gruhastha-minor-curse.json [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Gruhastha - Minor Curse",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/gruhastha-minor-curse.json:20]:
```suggestion
```

File: packs/data/boons-and-curses.db/gruhastha-moderate-curse.json [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+        },
+        "rules": [
+            {
+                "adjustment": {
+                    "criticalSuccess": "one-degree-worse",
+                    "failure": "one-degree-worse",
+                    "success": "one-degree-worse"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "label": "Gruhastha - Moderate Curse",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/gruhastha-moderate-curse.json:25]:
```suggestion
```

File: packs/data/boons-and-curses.db/hei-feng-minor-curse.json [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Hei Feng - Minor Curse",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/hei-feng-minor-curse.json:20]:
```suggestion
```

File: packs/data/boons-and-curses.db/hei-feng-moderate-curse.json [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "adjustment": {
+                    "failure": "one-degree-worse"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "label": "Hei Feng - Moderate Curse",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/hei-feng-moderate-curse.json:23]:
```suggestion
```

File: packs/data/boons-and-curses.db/kurgess-major-boon.json [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+            "value": 0
+        },
+        "rules": [
+            {
+                "adjustment": {
+                    "criticalFailure": "one-degree-better",
+                    "success": "one-degree-better"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "label": "Kurgess - Major Boon",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/kurgess-major-boon.json:24]:
```suggestion
```

File: packs/data/boons-and-curses.db/milani-major-boon.json [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Milani - Major Boon (Requests to satisfy Milani's edicts)",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/milani-major-boon.json:20]:
```suggestion
```

File: packs/data/boons-and-curses.db/shizuru-moderate-boon.json [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "diceNumber": 1,
+                "dieSize": "d6",
+                "key": "DamageDice",
+                "label": "Shizuru - Moderate Boon (Fire or Good damage)",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/shizuru-moderate-boon.json:22]:
```suggestion
```

File: packs/data/boons-and-curses.db/sivanah-minor-curse.json [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 0
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Sivanah - Minor Curse (Impersonate someone else)",
```
Comment (TikaelSol) [inline packs/data/boons-and-curses.db/sivanah-minor-curse.json:20]:
```suggestion
```

**Comment by stwlam** [comment]
Didn't this originally also remove the "feats?"

**Comment by LebombJames** [comment]
> Didn't this originally also remove the "feats?"

Nope, didn't touch the originals at all besides one of Iomdae's which had a few formatting errors

**Comment by stwlam** [comment]
> > Didn't this originally also remove the "feats?"
> 
> Nope, didn't touch the originals at all besides one of Iomdae's which had a few formatting errors

OK, we can nuke them after this is merged in.

**Pull request merged**